src/Proxy/SalesArticleProxy.cs

```diff
@@ -40,7 +58,7 @@ public IEnumerable<SalesArticle> Search(string searchTerm)
             }
 
             var json = new JsonSerializer();
-            return json.Deserialize<IEnumerable<SalesArticle>>(response.Value);
+            return json.Deserialize<IEnumerable<SalesArticle>>(response.Value).Take(20);
```

Comment:
what does the 20 limit do - is it to stop returning too much for generic searches?

Reply:
yes. stops it timing out if you type an 'a' and leave it for half a second.